service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/ClientInstallationServiceImpl.java

```diff
@@ -142,8 +142,8 @@ public void updateInstallation(
     @Override
     public void updateInstallation(Installation installationToUpdate, Installation postedInstallation) {
         // copy the "updateable" values:
-        installationToUpdate.setCategories(postedInstallation.getCategories());
```

Comment:
this is removed it was taken a Persistent set and overriding it with a normal set making hibernate merge the categories